projects/kubernetes/kube-master.yml

```diff
@@ -51,18 +51,19 @@ services:
      - /dev:/dev
      - /etc/resolv.conf:/etc/resolv.conf
      - /lib/modules:/lib/modules
+     - /run:/run:rshared,rbind
```

Comment:
Hmm, is `/var/run/secrets` actually a mount point? If it is not you shouldnt need the `rshared,rbind`, if it is then I guess so. Weird place to put a mountpoint if so...